common/libs/erc20.ts

```diff
@@ -57,7 +101,32 @@ const erc20Abi = [
         type: 'bool'
       }
     ]
+  },
+  {
+    name: 'Transfer',
```

Comment:
What's using the ERC20 `Transfer` event in this PR?

Reply:
Nothing really, just kept the ABI updated for future use since it can't hurt